src/main/resources/clientizen.mixins.json

```diff
@@ -7,6 +7,10 @@
   "client": [
     "ClientPlayNetworkHandlerMixin",
     "ClientWorldAccessor",
+    "EntityVisibilityMixin",
+    "IntPropertyAccessor",
+    "KeyBindingMixin",
+    "MinecraftClientMixin",
```

Comment:
This PR only adds one mixin, I believe the rest were accidently added in when resolving merge conflicts?

Reply:
Oops haha